draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1894,8 +1895,8 @@ The following procedure implements expand\_message\_xof.
 expand_message_xof(msg, DST, len_in_bytes)
 
 Parameters:
-- H, an extensible-output function.
-  H(m, d) hashes message m and returns d bytes.
+- H(m, d), an extendable-output function with input
+           message m and returning d bytes.
```

Comment:
```suggestion
- H(m, d), an extendable-output function that hashes input
           message m and returns d bytes.
```

Reply:
Well, feel free to replace "hash" with "process," even though I think hash is sufficient clear.

Reply:
done